### ISSUE TITLE
Single pointer control is not provided to adjust the column width of the columns in table:A11y_SQL Azure Data Studio_Query_Keyboard

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#26034]
Please do not close this bug. This bug should only be closed by Trusted Tester after veriﬁcation 
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug ﬁxing faster and easier." 

### GitHub Tags:
#A11yWCAG; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Nov2024; #SQL Azure Data Studio; #WCAG2.5.7;  #Win32; #Benchmark; #FTP; #Keyboard;  #Dragging Movements;#A11yWCAG2.2;

### Environment Details:
Application Name: SǪL Azure Data Studio 
Edition: Windows 11 Enterprise 
Version: 24H2 
Installed: 22-10-2024 
OS build: 26100.2033 
Experience: Windows Feature Experience Pack 1000.26100.23.0 

### Steps to Reproduce
1.Launch the application-Azure Database Project and install extension SǪL Database Projects. 
2.Tab to file and Hit Enter Key on new query
3.type and Run query. 
4.Tab to results tab and press enter.
5.Check using Single pointer control is provided to adjust the column width of the columns in table or not.

### Actual Results:
Single pointer control is not provided to adjust the column width of the columns in table

### Expected Results:
Single pointer control should be provided to adjust the column width of the columns in table

### 
### User Impact:
Some users cannot perform dragging movements in a precise manner. Others use a specialized or adapted input device, such as a trackball, head pointer, eye-gaze system, or speech-controlled mouse emulator, which may make dragging cumbersome and error-prone.
